README.md

```diff
@@ -66,3 +40,4 @@ Give a ⭐️ if you like this project!
 ## 📝 License
 
 This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] OPTIONAL: please make sure that all links you have in the readme file work. Currently, the license link is broken. This is happening because you did not add it to the project. Please consider adding the `MIT.md` license to the project root directory.